### PR TITLE
LondonClass7/AhmadJanah

### PR DESCRIPTION
**Pull request by AhmadJanah**
# Your Details

- Your Name:
- Your City:
- Your Slack Name:

# Homework Details

- Module:
- Week:

### CODE REVIEW

File: 1-exercises/H-array-methods-2/exercise2.js [hunk 1]
```diff
@@ -7,7 +7,9 @@
   Tip: use the string method .split() and the array method .join()
 */
 
-function capitalise(str) {}
+function capitalise(str) {
+  return [str.split("")[0].toUpperCase() , str.slice(1)].join("");
```
Comment (akbasmustafa) [inline 1-exercises/H-array-methods-2/exercise2.js:11]: Great work! You use always best practice array methods expressions.

File: 2-mandatory/7-password-validator.js [hunk 2]
```diff
@@ -25,8 +25,14 @@ PasswordValidationResult=  [false, false, false, false, true]
 */
 
 function validatePasswords(passwords) {
+    return passwords.map((pass, index) => containsUppercaseLetter(pass) && containsLowercaseLetter(pass) &&
+            containsNumber(pass) && containsSymbol(pass) && 
+            passwords.indexOf(pass) === index && pass.length >= 5);
```
Comment (akbasmustafa) [inline 2-mandatory/7-password-validator.js:30]: That's great. I learned best practices from your solution

Comment (AhmadJanah) [reply 2-mandatory/7-password-validator.js:30]: Thank you Mustafa.

**Review by iTrip23 — COMMENTED** [review]
Hi Ahmad,
Overall you did a great job, and I learned a few things from reading your code.
Keep up the good work and happy coding!

**Pull request closed without merging**